src/components/navbar/Navbar.jsx

```diff
@@ -0,0 +1,19 @@
+import { NAV_LOGO } from "src/utils/constants/constants";
+import Search from "src/components/search/Search";
+import "src/components/navbar/Navbar.component.scss";
+
+const Navbar = () => {
+  return (
+    <div className="navbar">
+      <div className="navbar-logo">
+        <img src={NAV_LOGO} alt="Logo" />
+        <h1>Todos</h1>
+      </div>
+      <div className="search-bar">
+        <Search />
+      </div>
```

Comment:
Use the div in search component
```suggestion
        <Search />
```